vaccine/application.py

```diff
@@ -1,40 +1,115 @@
-from typing import Optional
+from typing import List, Optional
 
 from vaccine.base_application import BaseApplication
+from vaccine.models import Message
 from vaccine.states import Choice, ChoiceState, EndState, ErrorMessage, FreeText
 
 
 class Application(BaseApplication):
-    START_STATE = "state_occupation"
+    START_STATE = "state_start"
+
+    async def process_message(self, message: Message) -> List[Message]:
+        if message.session_event == Message.SESSION_EVENT.CLOSE:
+            self.user.session_id = None
+            return [
+                message.reply(
+                    content="\n".join(
+                        [
+                            "We're sorry, but you've taken too long to reply and your "
+                            "session has expired.",
+                            "If you would like to continue, you can at anytime by "
+                            "typing the word *VACCINE*.",
+                            "",
+                            "Reply *MENU* to return to the main menu",
+                        ]
+                    ),
+                    continue_session=False,
```

Comment:
Does the framework allow for continuation?

Reply:
Yup, by default it continues session, this is to say that we want to end the session on our side (although it should already be ended because we're getting a message from the user that they've ended the session)